src/pipecat/frames/frames.py

```diff
@@ -706,6 +715,18 @@ def __str__(self):
         return f"{self.name}(pts: {pts}, text: [{self.text}], size: {self.size}, format: {self.format})"
 
 
+@dataclass
+class UserImageMessageFrame(SystemFrame):
+    """An image associated to a user."""
```

Comment:
This is the same description as given to `UserImageRawFrame`. What makes the `UserImageMessageFrame` different? Describe that.

Reply:
This is outdated. `UserImageMessageFrame` doesn't exist anymore.

Reply:
Well, I added it in this PR but then simplified a few things and just used `UserImageRequestFrame` and `UserImageRawFrame`.